ikarus/python/finiteelements/nonlinearelastic.hh

```diff
@@ -104,9 +104,9 @@ void registerNonLinearElastic(pybind11::handle scope, pybind11::class_<NonLinear
   cls.def(
       "calculateAt",
       [](NonLinearElastic& self, const FERequirements& req, const Dune::FieldVector<double, Traits::mydim>& local,
-         ResultType resType) {
-        if (resType == ResultType::PK2Stress)
-          return self.template calculateAt<ResultType::PK2Stress>(req, local);
+         std::string& resType) {
```

Comment:
This should by value or const reference. Mutablwe reference does not make sense here.